realm/realm-library/src/androidTest/assets/ios/README.md

```diff
@@ -105,46 +145,53 @@ RLM_ARRAY_TYPE(AllTypes)
     [realm addObject:obj];
     [realm commitWriteTransaction];
 
-    [AppDelegate setDefaultRealm: [NSString stringWithFormat:@"%@-alltypes-min.realm", version]];
-    realm = [RLMRealm defaultRealm];
+    realm = [AppDelegate appDefaultRealm:[NSString stringWithFormat:@"%@-alltypes-min.realm", version]];
     [realm beginWriteTransaction];
     obj = [[IOSAllTypes alloc] init];
     obj.boolCol = FALSE;
     obj.shortCol = SHRT_MIN;
     obj.intCol = INT_MIN;
     obj.longCol = LONG_MIN;
-    obj.floatCol = FLT_MIN;
-    obj.doubleCol = DBL_MIN;
+    obj.longLongCol = std::numeric_limits<int64_t>::min();
+    obj.floatCol = -FLT_MAX;
+    obj.doubleCol = -DBL_MAX;
     obj.byteCol = [NSData dataWithBytes:no_bytes length:sizeof(no_bytes)];
     obj.stringCol = @"";
-    obj.dateCol = [NSDate dateWithTimeIntervalSince1970: 0];
+    obj.dateCol = [NSDate dateWithTimeIntervalSinceReferenceDate:-DBL_MAX];
     [realm addObject:obj];
     [realm commitWriteTransaction];
+    NSLog(@"%@, obj.longLongCol : 0x%llx\n",[NSString stringWithFormat:@"%@-alltypes-min.realm", version], obj.longLongCol);
```

Comment:
I'd suggest to leave debug output until the new datetime is released. Without one, it is very natural to expect to get the exactly the same value we put in. _We_ know that's not the case but still...